pom.xml

```diff
@@ -82,7 +82,7 @@
     <hadoop1.version>1.2.1</hadoop1.version>
     <cascading.version>2.7.1</cascading.version>
     <cascading3.version>3.1.2</cascading3.version>
-    <parquet.format.version>2.7.0</parquet.format.version>
+    <parquet.format.version>2.8.0-SNAPSHOT</parquet.format.version>
```

Comment:
Referencing a snapshot dependency is not allowed on master and won't work anyway. After releasing the new parquet-format it will be available under the version number `2.8.0`. Until then, we won't be able to push this change.

Reply:
Bumped to 2.8.0 in this patch. Hopefully not an issue that it's not in a separate patch.

Reply:
2.8.0 was mostly released for byte_stream_split so I think it's fine to upgrade the version in this patch. :)